hudi-common/src/test/java/org/apache/hudi/common/util/TestSortUtils.java

```diff
@@ -77,7 +78,7 @@ record = new HoodieAvroRecord(new HoodieKey("record1", "partition1"), payload);
       record = new TestSparkRecord(new HoodieKey("record1", "partition1"), payload);
     }
     String[] userSortColumns = new String[] {"non_pii_col", "timestamp"};
-    FlatLists.ComparableList<Comparable<HoodieRecord>> comparableList = SortUtils.getComparableSortColumns(record, userSortColumns, Schema.parse(SCHEMA), suffixRecordKey, true);
+    FlatLists.ComparableList<Comparable<HoodieRecord>> comparableList = SortUtils.getComparableSortColumns(record, userSortColumns, HoodieSchema.parse(SCHEMA), suffixRecordKey, true);
```

Comment:
Nit: We can use HoodieSchema to parse the `SCHEMA` string to a `HoodieSchema`, after which, we can remove the importation of `org.apache.avro.Schema`.